@uportal/action-favorites/src/i18n.js

```diff
@@ -0,0 +1,62 @@
+import Vue from "vue";
+import VueI18n from "vue-i18n";
+
+const locales = {
+  en: {
+    message: {
+      favorites: {
+        add: "Add to favorites",
+        remove: "Remove from favorites"
+      }
+    }
+  },
+  fr: {
+    message: {
+      favorites: {
+        add: "Ajouter aux favoris",
+        remove: "Supprimer des favoris"
+      }
+    }
+  }
+};
+
+function checkLang() {
+  // These window.navigator contain language information
+  // 1. languages -> Array of preferred languages (eg ["en-US", "zh-CN", "ja-JP"]) Firefox^32, Chrome^32
+  // 2. language  -> Preferred language as String (eg "en-US") Firefox^5, IE^11, Safari,
+  //                 Chrome sends Browser UI language
+  // 3. browserLanguage -> UI Language of IE
+  // 4. userLanguage    -> Language of Windows Regional Options
+  // 5. systemLanguage  -> UI Language of Windows
```

Comment:
Could https://github.com/i18next/i18next-browser-languageDetector help with detection?
Or could this be moved into it's own npm module?
I'm cautious about the potential of this being copied and pasted into each vue component.